simplified-app-ekirjasto/src/main/assets/txnative/sv/txstrings.json

```diff
@@ -1407,17 +1674,20 @@
       "settingsPasswordShow" : {
          "string" : "Visa lösenord"
       },
+      "settingsPlaceholder" : {
+         "string" : "Lorem ipsum dolor sit amet, probatus volutpat has at, vis adhuc iuvaret omittantur an, sadipscing conclusionemque his ad. Eam ut simul tempor. Primis consetetur appellantur vim eu, eum an iudico dolorum. Ad vis utamur honestatis, sanctus debitis referrentur an eam."
+      },
       "settingsPrivacy" : {
          "string" : "Integritetspolicy"
       },
       "settingsPub" : {
-         "string" : ""
+         "string" : "Utgivare"
```

Comment:
This should be shorter, the English is "Pub" and the Finnish is "Julk". Being this long might break the UI. Should be fixed in Transifex.

Reply:
Chosen translation: "Utg"